scripts/set-version.sh

```diff
@@ -15,4 +15,4 @@ if [ $# != 1 ] || [ "$1" = "?" ] || [ "$1" = "--help" ]; then
     exit 1
 fi
 
-sed -i -e "s;\(^TAG?*=\|intel/crypto-perf:\|intel/intel-[^ ]*:\)[^ ]*;\1$1;g" $(git grep -l '^TAG?*=\|intel/crypto-perf:\|intel/intel-[^ ]*:' Makefile deployments)
+sed -i -e "s;\(^TAG?*=\|intel/crypto-perf:\|intel/opae-nlb:\|intel/intel-[^ ]*:\)[^ ]*;\1$1;g" $(git grep -l '^TAG?*=\|intel/crypto-perf:\|intel/opae-nlb:\|intel/intel-[^ ]*:' Makefile deployments demo/intelfpga-job.yaml)
```

Comment:
do you need to update here as well to `opae-nlb-demo` ?

Reply:
yes, sorry. fixed.